assets/src/components/v2/on_bus/no_service.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ComponentType } from "react";
+import Header from "../departures/header";
+
+const DeparturesNoService: ComponentType = () => {
+  return (
+    <div className="departures-container">
+      <Header {...{ title: "CONNECTIONS", arrow: null }} />
```

Comment:
I think this should work and would be a more conventional way of expressing this:

```tsx
<Header title="CONNECTIONS" arrow={null} />`
```